consensus/src/simplex/signing_scheme/bls12381_threshold.rs

```diff
@@ -85,6 +88,12 @@ impl<P: PublicKey, V: Variant> Scheme<P, V> {
     /// * `share` - local threshold share for signing
     pub fn new(participants: Ordered<P>, polynomial: &Public<V>, share: Share) -> Self {
         let identity = *poly::public::<V>(polynomial);
+        let quorum = quorum(participants.len() as u32);
```

Comment:
If you import `OrderedExt` you can also just do `participants.quorum()`.

Reply:
ooooo